### ISSUE TITLE
[Cata][Talends][Druid] Shooting stars

### ISSUE THREAD

**Issue by MoireMoire** [NRG-Wow/NRGWow-Bugtracker#149]
**Reporter(Player/Character) Name**: Moire

**Description**: Shooting stars issue

**Current behaviour**: Doesn't always reset the cooldown, sometimes i have to wait until the normal cooldown expires, and no subsequent proc happens till then (which might reset the remaining cooldown). Usually it happens when i get two procs pretty close to eachohter, mainly when i got two or more targets dotted

**Expected behaviour**: Should reset the cooldown instantly

**Steps to reproduce the problem**: 

1.  multidot two or three targets
2. get two subsequent procs
3. no reset will happen on the second (ofc not all the times)

**Comment by Wilcalaf** [comment]
So you get the cast reduction and not the cooldown?
i dont seem to be able to reproduce it.
so far everytime i get the proc the cooldown resets
tested on 2 targets, 3 targets , 4 targets and 1 targets.
gonna keep trying for a while

---- after 200+ procs. it didnt happen.
need more details on this.
Where did this happen? on which mobs? inside a dungon?

**Comment by MoireMoire** [comment]
Actually i need to check on that, will give more infos in a couple hours

Il ven 10 ago 2018, 09:01 Wilcalaf <notifications@github.com> ha scritto:

**Comment by MoireMoire** [comment]
Actually happened both in dungeons and raids

**Comment by Wilcalaf** [comment]
So you see the buff on yourself (Shooting stars)
and the action bar glowing but the spell cooldown stil still up right?

**Comment by MoireMoire** [comment]
Exactly, i have to wait the full cooldown, which is 15 seconds, because when this happens i seem to not be able to get other procs. I've been trying to do some dummy test myself and there seems to work fine, i don't really know

**Comment by MoireMoire** [comment]
I will try to remember and make a SS next time it happens, i'll wait till about half the cooldown.

**Comment by virpyre** [comment]
I tested Moonfire & Insect Swarms in Orgrimmar on 3 Raider's Training Dummies at the same time. I had 8 Shooting Star proc's but not one of them reset the CD on Starsurge.

From what I can tell is Shooting Star lasts for 12 seconds or until you use Starsurge. It also only has a 4% chance when casting Moonfire or Insect Swarm to reset Starsurge by 100%.

**Comment by MoireMoire** [comment]
That's a fact, what i was reporting is that sometimes i'm not able to use that proc.

**Comment by Wilcalaf** [comment]
Im still not able to reproduce the problem. Can you please write here next time it happens to you exactly where and which fight/boss/add you were fighting and how many?

**Comment by MoireMoire** [comment]
Will do.
